internal/controller/state/graph/grpcroute.go

```diff
@@ -444,3 +456,26 @@ func validateGRPCHeaderMatch(
 
 	return allErrs
 }
+
+func checkForUnsupportedGRPCFields(rule v1.GRPCRouteRule, rulePath *field.Path) field.ErrorList {
+	var ruleErrors field.ErrorList
+
+	if rule.Name != nil {
+		ruleErrors = append(ruleErrors, field.Forbidden(
+			rulePath.Child("name"),
+			"NGINX Gateway Fabric does not support \"SectionName\" field at the moment",
+		))
+	}
+	if rule.SessionPersistence != nil {
+		ruleErrors = append(ruleErrors, field.Forbidden(
+			rulePath.Child("sessionPersistence"),
+			"NGINX Gateway Fabric does not support \"SessionPersistence\" field at the moment",
```

Comment:
Similarly, let's remove the NGINX Gateway Fabric and "at the moment".

Technically we don't need the name of the field in the error message either, as the error will be reported on the appropriate field as given in line 471, but specifying it again is consistent with other areas of the codebase so we can leave it in:

```suggestion
			"sessionPersistence is not supported",
```